src/parser/monk/mistweaver/modules/talents/Lifecycles.tsx

```diff
@@ -78,15 +78,15 @@ class Lifecycles extends Analyzer {
     // Checking for chiji stacks and determine mana reduction
     const chijiStacksAtEnvCast = this.selectedCombatant.getBuff(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_BUFF.id, event.timestamp)?.stacks;
     if (!chijiStacksAtEnvCast) {
-      this.calculateEnvManaSaved(SPELLS.ENVELOPING_MIST.manaCost);
+      this.calculateEnvManaSaved(SPELLS.ENVELOPING_MIST.manaCost ? SPELLS.ENVELOPING_MIST.manaCost : 0);
       return;
     }
     //check for free cast from chiji
     if (chijiStacksAtEnvCast === MAX_CHIJI_STACKS) {
       return;
     }
     //have to do this weird because blizzard decided to make each chiji stack reduce the mana cost by 1001 instead of and exact 33%
-    const modifiedManaCost = SPELLS.ENVELOPING_MIST.manaCost - (CHIJI_MANA_SAVED_PER_STACK * chijiStacksAtEnvCast);
+    const modifiedManaCost = (SPELLS.ENVELOPING_MIST.manaCost ? SPELLS.ENVELOPING_MIST.manaCost : 0) - (CHIJI_MANA_SAVED_PER_STACK * chijiStacksAtEnvCast);
```

Comment:
If these values are ever null and it defaults to 1 it will just provide inaccurate information making the whole statistic pointless and misleading. Since it wouldn't crash people won't come ask for it to be fixed either and just assume damn lifecycles is literally the worst talent in the game and even playing it to the mas it doesn't save me more than 100 mana on top of then maybe going into peak and making me have to deal with them there